index.html

```diff
@@ -84,14 +84,151 @@ <h1>Hey there.<span class="span__br"><br /></span> I’m Mateo.
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
     </div>
+  </section>
+
+  <!-----About Me--------->
+
+  <section id="about">
+    <div class="about_me">
+      <div class="about_container">
+        <h2>About me</h2>
+
+        <p class="about_description">
+          I can help you build a product , feature or website
+           Look through some of my work <span class="span__br2"><br /></span> 
+           and experience! If you like what you see and have a project 
+           you need coded, don’t hesitate to contact me.
+        </p>
+
+        <button class="about_button" type="button">
+          <a href="#">Get my resume</a>
+        </button>
+      </div>
+
+      <div class="about_wrapper">
+        <article class="about_me_container languages">
+          <div class="about_languages">
+            <div class="about_icon">
+              <img
+                src="./assets/icons/LanguageIcon.svg"
+                />
+            </div>
+
+            <h3 class="about_title">Languages</h3>
+
+            <ul class="about_cardtags">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+        </article>
+
+        <article class="about_me_container frameworks">
+          <div class="about_languages">
+            <div classs="about_icon">
+              <img
+              src="./assets/icons/FrameworksIcon.svg"
+              />
+            </div>
+
+            <h3 class="about_title">Frameworks</h3>
 
+            <ul class="about_cardtags">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </article>
+
+        <article class="about_me_container skills">
```

Comment:
- [ ] From the images below you will notice that your `about me` text does not have the same color as the provided figma design. please make the necessary changes to this section :+1: 
- [ ] Also the circled red part which has the image for the about me is covering the text and this is making your design inconsistent with the figma design please i kindly suggest you make the appropriate change to this part of your design :+1: :sparkles: 
- figma design
![xf](https://github.com/mateoburvi/Portfolio/assets/89055502/74c07860-fe7f-41bf-b0bf-f4cafc946caa)
- your design
![fx](https://github.com/mateoburvi/Portfolio/assets/89055502/c502126b-d63c-4d6d-9fc1-8c77b00b4a4f)